ansys/mapdl/core/pool.py

```diff
@@ -33,9 +33,11 @@ def available_ports(n_ports, starting_port=MAPDL_DEFAULT_PORT):
     return ports
 
 
-class LocalMapdlPool():
+class LocalMapdlPool:
     """Create a pool of MAPDL instances.
 
+    Requires MAPDL 2020 R2 or later.
```

Comment:
```suggestion
    .. note::
       Requires MAPDL 2020 R2 or later.
```

Using the note directive makes it pop-up.